website_anchor_smooth_scroll/__manifest__.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antiun Ingeniería S.L. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Smooth Scroll for Website Anchors',
+    'summary': 'Replace default behavior for internal links (anchor only) with'
+               ' smooth scroll',
+    'version': '11.0.1.0.0',
+    'category': 'Website',
+    'website': 'http://www.antiun.com',
```

Comment:
@pedrobaeza need I put oca website ?

Reply:
New rule is to put github repository url (https://github.com/OCA/website)